### ISSUE TITLE
Inconsistent server responding time due to workstation network issues

### ISSUE THREAD

**Issue by thomasinfor** [NTU-SP/SP-Hw1-release#49]
Lots of us have suffered resource exhaustion on linux1, and as mentioned in #43, server response time may become delayed and unpredictable.

I'm wondering if our server will be run directly on linux1 or not? Will the same issue happen when TAs grade our code?

**Comment by imingasco** [comment]
No. We will check the status of linux1 before grading your homework.
